<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.springframework.test.context.junit4.statements.SpringFailOnTimeoutTests" tests="5" skipped="0" failures="0" errors="0" timestamp="2020-08-26T04:38:55" hostname="PC-20191023THJJ" time="0.056">
  <properties/>
  <testcase name="negativeTimeout" classname="org.springframework.test.context.junit4.statements.SpringFailOnTimeoutTests" time="0.0"/>
  <testcase name="timeoutExceptionThrownIfNoUserException" classname="org.springframework.test.context.junit4.statements.SpringFailOnTimeoutTests" time="0.052"/>
  <testcase name="nullNextStatement" classname="org.springframework.test.context.junit4.statements.SpringFailOnTimeoutTests" time="0.0"/>
  <testcase name="userExceptionPropagates" classname="org.springframework.test.context.junit4.statements.SpringFailOnTimeoutTests" time="0.003"/>
  <testcase name="noExceptionThrownIfNoUserExceptionAndTimeoutDoesNotOccur" classname="org.springframework.test.context.junit4.statements.SpringFailOnTimeoutTests" time="0.001"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
